src/Configuration/src/ConfigServerBase/ConfigServerConfigurationProvider.cs

```diff
@@ -834,7 +828,6 @@ protected internal async void RefreshVaultTokenAsync(object state)
             }
             finally
             {
-                client.Dispose();
```

Comment:
Does this client get disposed elsewhere? I see a lot of dispose removal (and the comment about "after token") but am missing where they ultimately get cleaned.

Reply:
With this change, the client lives until the app shuts down